java/src/main/java/com/google/cloud/dataproc/templates/databases/README.md

```diff
@@ -236,3 +236,49 @@ export REGION=us-central1
 ```
 
 Note-: ```mongo.gcs.output.mode=<append|overwrite|ignore|errorifexists>``` is used to specify the mode for output in GCS. It has a default value of ```append``` if nothing is passed.
+
+# Mongo To BigQuery
+
+Template for exporting a MongoDB collection to a BigQuery table.
+
+Required JAR files
+
+It uses the [MongoDB Spark Connector](https://www.mongodb.com/products/spark-connector) and [MongoDB Java Driver](https://jar-download.com/?search_box=mongo-java-driver) for reading data from MongoDB Collections. To write to BigQuery, the template needs [Spark BigQuery Connector](https://github.com/GoogleCloudDataproc/spark-bigquery-connector).
+
+This template has been tested with the following versions of the above mentioned JAR files:
+
+1. MongoDB Spark Connector: 2.12-2.4.0
+2. MongoDB Java Driver: 3.9.1
+3. Spark BigQuery Connector: 2.12
+
+Arguments
+
+- `mongo.bq.input.uri`: MongoDB Connection String as an Input URI (format: `mongodb://host_name:port_no`)
+- `mongo.bq.input.database`: MongoDB Database Name (format: Database_name)
+- `mongo.bq.input.collection`: MongoDB Input Collection Name (format: Collection_name)
+- `mongo.bq.output.dataset`: BigQuery dataset id (format: Dataset_id)
+- `mongo.bq.output.table`: BigQuery table name (format: Table_name)
+- `mongo.bq.temp.bucket.name`: Cloud Storage bucket name to store temporary files (format: Bucket_name)
+- `mongo.gcs.output.mode`: Output write mode (one of: append,overwrite,ignore,errorifexists)
```

Comment:
#1. The output modes as case sensitive ?
#2. Ideally it should matches the example below

Reply:
Modes are in lower case. [Spark Document](https://spark.apache.org/docs/latest/sql-data-sources-load-save-functions.html#save-modes)

Reply:
Only `append` and `overwrite` are supported.